packages/app-page-builder/src/pageEditor/config/blockEditing/BlockPreview.tsx

```diff
@@ -46,6 +45,7 @@ const BlockPreview: React.FC<BlockPreviewProps> = props => {
                 {onDelete && (
                     <Styled.DeleteBlock>
                         <ConfirmationDialog
+                            style={{ zIndex: 99 }}
```

Comment:
Why this change?

Reply:
We need this change because `ConfirmationDialog` and `EditBlockDialog` were rendering below `All Blocks` component, and because of that the user couldn't see those modals and couldn't interact with them.

<img width="1800" alt="Знімок екрана 2023-07-11 о 10 36 17" src="https://github.com/webiny/webiny-js/assets/129154672/43133099-dbae-4a40-a613-15abed5d873b">

Reply:
Why not just do 100? Why 99? 🤔 

Can we add a comment into the code, just so one day when we get back, know why this is here?